spec/support/graphql/response_parser.rb

```diff
@@ -1,11 +1,18 @@
 module GraphQL
   module ResponseParser
-    def execute_and_parse_graphql_response(query:, variables:)
-      result = FocusSchema.execute(
-        query,
-        variables: { input: variables }
-      )
-      result.to_h.delete('data').with_indifferent_access
+
+    attr_accessor :parse_graphql_response
+
+    def execute_and_parse_graphql_response(query:, variables:nil, current_user: nil)
+      params = {query: query, context: { current_user: current_user }}
+      params.merge!({ variables: variables }) if variables.present?
+      result = FocusSchema.execute(params)
+      @parse_graphql_response = 
```

Comment:
actually, do you think we can use this parsed response not only test, for API response for frontend? I'm trying to get rid of the `data` it is so annoying that I need to restructure for every single queries, otherwise I have to call `data.currentUser` like this. I've tried on the client but doesn't work at all so it would be nice if it's possible on backend 
<img width="655" alt="Screen Shot 2021-09-06 at 2 31 13 PM" src="https://user-images.githubusercontent.com/49046928/132261622-d875cf3c-3024-4d9e-92fd-39ee1f977140.png">

<img width="532" alt="Screen Shot 2021-09-06 at 2 30 31 PM" src="https://user-images.githubusercontent.com/49046928/132261621-63df01b3-221d-40d9-8561-2d6a1b41d64b.png">

Reply:
it's possible :), I can include it my PR or you can include it if you want 
<img width="883" alt="Screen Shot 2021-09-06 at 2 59 14 PM" src="https://user-images.githubusercontent.com/49046928/132262630-5b9c2537-be15-4b79-8262-f1187a0614cd.png">

Reply:
I don't think we change this one.  This custom result can be risky and get potential issue because this only supports correct cases. So It's good to use `data` wrapper in frontend.